.github/workflows/test.yaml

```diff
@@ -7,3 +7,15 @@ jobs:
   unit-tests:
     uses: canonical/operator-workflows/.github/workflows/test.yaml@main
     secrets: inherit
+  look-for-outdated-code:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install dependencies
+        run: |
+          sudo apt update && sudo apt install python3-pip
+          pip3 install pyupgrade refurb
+      - name: Check for any code improvements
+        run: |
+          find . -name '*.py' | xargs pyupgrade --py310-plus
```

Comment:
I think pyupgrade accepts multiple filenames so a command substitution will be enough, like `pyupgrade --py310-plus $(find . -name '*.py')`. Same for refurb.

Reply:
Thank you, fixed